Capri.Web/Configuration/MappingProfile.cs

```diff
@@ -10,6 +10,7 @@ public class MappingProfile : Profile
         public MappingProfile()
         {
             CreateMap<PromoterUpdate, Promoter>();
+            CreateMap<PromoterRegistration, Promoter>();
```

Comment:
I think it's a good moment for that separation I've talked on previous planning. :)